hail/python/hailtop/batch_client/aioclient.py

```diff
@@ -462,14 +474,21 @@ async def wait(self,
         with BatchProgressBar(disable=disable_progress_bar) as progress2:
             return await self._wait(description, progress2, disable_progress_bar, starting_job)
 
-    async def debug_info(self):
+    async def debug_info(self,
+                         _job_filter: Optional[Callable[[JobListEntry], bool]] = None,
```

Comment:
Just a drive by comment: You can use the query string `q` to search for failed jobs instead of filtering client-side

Reply:
done